docs/references/javascript/organization-domain.mdx

```diff
@@ -0,0 +1,167 @@
+        },
+        {
+            cells: [
+                <code>affiliationEmailAddress</code>,
+                <code>string | null</code>,
+                "The email address that was used to verify this organization domain organization.",
+            ]
+        },
+        {
+            cells: [
+                <code>totalPendingInvitations</code>,
+                <code>number</code>,
+                "The number of total pending invitations sent to emails that match the domain name.",
+            ]
+        },
+        {
+            cells: [
+                <code>totalPendingSuggestions</code>,
+                <code>number</code>,
+                "The number of total pending suggestions sent to emails that match the domain name.",
+            ]
+        },
+        {
+            cells: [
+                <code>createdAt</code>,
+                <code>Date</code>,
+                "Date of the time the membership was created.",
+            ]
+        },
+        {
+            cells: [
+                <code>updatedAt</code>,
+                <code>Date</code>,
+                "Date of the last time the membership was updated.",
+            ]
+        },
+    ]}
+/>
+
+### `OrganizationDomainVerification`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `status` | `'unverified' \| 'verified'` | The status of the verification process. |
+| `strategy` | `email_code` | A string that indicates strategy of the verification. |
+| `attempts` | number | A number that indicates how many attempts have occured in order to verify the domain. |
+| `expiresAt` | `Date` | The expiration date and time of the verification. |
+
+
+## Methods
+
+### `delete()`
+
+```typescript
+function delete: () => Promise<void>;
+```
+
+Delete the organization domain and removes it from the organization.
+
+
+## `prepareAffiliationVerification()`
+
+```typescript
+function prepareAffiliationVerification(params: PrepareAffiliationVerificationParams): Promise<OrganizationDomain>;
+```
+
+Begins the verification process of a created organization domain. This is a required step in order to complete the registration of the domain under the organization.
+
+### `PrepareAffiliationVerificationParams`
+
+<Tables
+    headings={["Name", "Type", "Description"]}
+    headingsMeta={[{maxWidth:'300px'},{maxWidth: '300px'},{maxWidth: '300px'}]}
+    rows={[
+        {cells: [<code>affiliationEmailAddress</code>, <code>string</code>, <>An email address that is affiliated with the domain name (e.g. user@example.com). </>]},
+    ]}
+/>
+
+### Returns
+
+<Tables
+    headings={["Type", "Description"]}
+    headingsMeta={[{}, {maxWidth: "300px"}]}
+    rows={[
+        {cells: [<code>Promise{"<"}<a href="/docs/references/javascript/organization-domain">OrganizationDomain</a>{">"}</code>, <>This method returns a <code>Promise</code> which resolves with a <code><a href="/docs/references/javascript/organization-domain">OrganizationDomain</a></code> object.</>]}

```

Comment:
We don't need to link to Organization Domain docs, because the user is already on this page.

```suggestion
        {cells: [<code>Promise{"<"}OrganizationDomain{">"}</code>, <>This method returns a <code>Promise</code> which resolves with a <codeOrganizationDomain</code> object.</>]}

```